docker/kubectl.version

```diff
@@ -1,5 +1,4 @@
-KUBECTL_VERSION=v1.13.8
-KUBECTL_CHECKSUM_amd64=b24d2f026064945a7e1571fd413bd974e165ca4514e35a3dae7df8cbf97bd283169c40e7720cbf40d6fdce2b37483df8a02a58c3a9cc9fc08aa386b43a467c0a
-KUBECTL_CHECKSUM_arm=dc60bdf00e6c7806e3c11e4f73e2ff27a603e968f22567d8c87ed5ece04263e557cb0df8d7b5196bc28a96b0b1f8ae104c503e9b9d90ce77c25e85954b54e178
-KUBECTL_CHECKSUM_arm64=00a98acd51107d1cb935cfc07ca31487290412f92ac34a91ec8c7f4b802bf798a7cc9cac22978a92001641c3c99a651c7f9a9f7f64f7cc4839a9d26021667a3b
-
+KUBECTL_VERSION=v1.14.9
```

Comment:
Is the version bump intentional?

Reply:
Yes, it even has a dedicated commit (`5cccc11`). The previous version did not support `kubectl apply -k`.